sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -399,15 +399,15 @@ def _get_tags(self):
         return tags
 
 
-class RequiresPositiveYRegressor(LinearRegression):
+class RequiresPositiveRegressor(LinearRegression):
```

Comment:
Can we define another `RequiresXPositiveRegressor` and use that to check only the positive X case? This way, `RequiresPositiveYRegressor` can still be used to check only the positive y case.